exporter/collector/integrationtest/testdata/fixtures/ops_agent_self_metrics.json

```diff
@@ -10,31 +10,25 @@
                         }
                     },
                     {
-                        "key": "host.name",
+                        "key": "net.host.name",
                         "value": {
                             "stringValue": "ops-agent-metrics.c.otel-opsagenttester.internal"
                         }
                     },
                     {
-                        "key": "job",
```

Comment:
@dashpole I'm assuming you are intentionally removing all the `job` keys rather than switch to `service.name` (trying to gleam context from https://github.com/open-telemetry/opentelemetry-specification/pull/2381). Are these obsolete now or will be added back later? You do have a new `service.name` entry in the workload_metrics fixture below so just curious

Reply:
Good catch.  I'm trying to remove the job label everywhere.  That one below should've been removed, rather than changed.